arrayMethods_lab.js

```diff
@@ -0,0 +1,291 @@
+// # Exercises
```

Comment:
Great job. I see that you are understanding the array methods! Keep up the great work. You should approach the bonus questions! But remember to create functions.